test/development/sass-error/index.test.ts

```diff
@@ -0,0 +1,43 @@
+import { nextTestSetup } from 'e2e-utils'
+import { getRedboxSource } from 'next-test-utils'
+
+describe('app dir - css', () => {
+  const { next, skipped } = nextTestSetup({
+    files: __dirname,
+    skipDeployment: true,
+    dependencies: {
+      sass: 'latest',
+    },
+  })
+
+  if (skipped) {
+    return
+  }
+
+  describe('sass support', () => {
+    ;(process.env.TURBOPACK ? describe : describe.skip)(
```

Comment:
What's about webpack? Could we only provide a different snapshot for it?

Reply:
webpack css-loader does not report any error for this case.